bemoji

```diff
@@ -20,6 +20,14 @@ bm_private_mode=${BEMOJI_PRIVATE_MODE:-false}
 # Do not sort results
 bm_ignore_recent=${BEMOJI_IGNORE_RECENT:-false}
 
+declare -A default_pickers=(
```

Comment:
I am not entirely sure which way would be the best to sort pickers:
I thought about going from most well known (e.g. most GH stars) to least well known, but that seems counter-productive since most people will often want to use the most 'obscure' picker they have installed I would think?
However, going the reverse way also seems counter-intuitive.

I have left it as is for now and just appended ilia, let me know if you have a better idea or some input on this.

Reply:
What about passing a picker via flag? I think by default whatever order is fine, but as for which one to actually execute, it makes most sense to me to allow the user to pass that in, and an argument to the script seems cleanest, even over an `ENV` var.

Reply:
I think you are right and we can probably actually combine both worlds by letting users choose a default picker or supply a custom one through the same interface. I'll open a new issue for that since it goes a bit beyond this PR.

Reply:
And just as a last note to myself: the order of initializing the hash map does not even matter in bash, it's always sorted based on hash in the end.

Reply:
Looks like tessen [does something similar](https://github.com/ayushnix/tessen/blob/79f05b8ca5f31d7de81b37db3fecaf599b2edb12/tessen#L547), FIFO-ish.